opentelemetry-api/src/opentelemetry/context/async_context.py

```diff
@@ -12,32 +12,34 @@
 # See the License for the specific language governing permissions and
```

Comment:
I think this file was changed by mistake? Could you please remove these changes from the PR?

Reply:
Not, it wasn't.
This file was changed in purpose to make it "importable" for Python <3.7.
Please see a [corresponding build](https://travis-ci.org/Jamim/opentelemetry-python/builds/595431924). That build failed with:
```
==================================== ERRORS ====================================
_ ERROR collecting opentelemetry-api/src/opentelemetry/context/async_context.py _
ImportError while importing test module '/home/travis/build/Jamim/opentelemetry-python/opentelemetry-api/src/opentelemetry/context/async_context.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
opentelemetry-api/src/opentelemetry/context/async_context.py:16: in <module>
    from contextvars import ContextVar
E   ModuleNotFoundError: No module named 'contextvars'
```

Reply:
But doesn't this break the logic of the importing `__init__.py`? You are not supposed to import that file directly anyway.

Reply:
> But doesn't this break the logic of the importing __init__.py?
No, it doesn't.

<details>
<summary>Python 3.7</summary>

```
$ python
Python 3.7.3 (default, Apr  3 2019, 19:16:38) 
[GCC 8.0.1 20180414 (experimental) [trunk revision 259383]] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from opentelemetry.context import Context
>>> Context
AsyncRuntimeContext({})
```
</details>

<details>
<summary>Python 3.6</summary>

```
$ python
Python 3.6.8 (default, Oct  7 2019, 12:59:55) 
[GCC 8.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from opentelemetry.context import Context
>>> Context
ThreadLocalRuntimeContext({})
```
</details>

> You are not supposed to import that file directly anyway.

Having non-importable files in the project is not good in general.